OpenRA.Mods.Common/Traits/Render/WithHarvestAnimation.cs

```diff
@@ -65,5 +70,13 @@ public static int ZOffsetFromCenter(Actor self, WPos pos, int offset)
 			var delta = self.CenterPosition - pos;
 			return delta.Y + delta.Z + offset;
 		}
+
+		public IEnumerable<IRenderable> ModifyRender(Actor self, WorldRenderer wr, IEnumerable<IRenderable> r)
+		{
+			if (string.IsNullOrEmpty(info.Palette))
+				return r;
+			else
+				return r.Where(a => a.IsDecoration).Select(a => a.WithPalette(wr.Palette(info.Palette))).Concat(r.Where(a => !a.IsDecoration));
```

Comment:
This will replace the palette of all decorative animations, which is a giant hack.